README.md

```diff
@@ -18,7 +28,23 @@ Additionally, libraries to interface with Warp (will) exist in JavaScript (TypeS
 
 #### Linux
 
-***TBD***
+**Ubuntu WSL (Maybe also Ubuntu + Debian)**
+| Dep  | Install Command                                                  |
+|------|------------------------------------------------------------------|
+| Rust | `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs \| sh` |
+| Build Essentials | `sudo apt install build-essential` |
+| CMake | `sudo apt install cmake` |
+| LLVM libs & headers | `sudo apt install llvm-dev` |
+| udev libs & headers | `sudo apt install libudev-dev` |
```

Comment:
Does blink require udev from what you know of?

Reply:
Don't know to be honest. I just C&P from the CI action file.